scripts/feature_tests/upgrade/upgrade_vars.sh

```diff
@@ -3,20 +3,23 @@ set -eux
 
 # Folder created for specific capi release when running
 # ${CLUSTER_API_REPO}/cmd/clusterctl/hack/create-local-repository.py
-# For CAPI release v0.3.12, the folder v0.3.8 is created
 
-export CAPIRELEASE_HARDCODED="v0.3.8"
+#export CAPIRELEASE_HARDCODED="v0.3.8"
```

Comment:
uncomment this. When upgrading capi within` (v0.4.0, v0.4.1 and v0.4.2)`, the value should be `v0.4.99`

Reply:
Done.